frontend/actions/userActions.js

```diff
@@ -180,9 +180,20 @@ export function signIn (email, password) {
         password,
         username: email // Passport needs this property to be called username!
       }
-    }).then(user => {
-      dispatch(setRemoteUser(user))
-    }).catch(response => {
+    })
+    .then(user => {
+      getNewBearerToken(user).then(
+        token => console.log({token})
+      ).catch(
+        err => console.error({err})
+      )
+
+      return user
+    })
+    .then(user => dispatch(setRemoteUser(user)))
+    .catch(response => {
+      console.log(response)
+      console.error(response.error)
```

Comment:
Remove logs

Reply:
Of course, just want it to work and be finished / ready first

Reply:
Okay - I see the in-progress label now 👍